channels/yandex-alice/src/main/kotlin/com/justai/jaicf/channel/yandexalice/api/AliceApi.kt

```diff
@@ -53,7 +53,9 @@ class AliceApi(
 
     fun getImageId(url: String) = images.getOrPut(url) { uploadImage(url).id }
 
-    fun uploadImage(url: String): Image = runBlocking {
+    fun getImageUrl(id: String) = images.entries.find { id == it.value }?.key
```

Comment:
I suggest making this api method internal. It's used only for our internal logic.

Reply:
Good suggestion